uwblueprint/social-diversity-for-children
# Setup Prisma ORM and Local Database

### Notion ticket link



[Set & Hook up database](https://www.notion.so/uwblueprintexecs/Set-Hook-up-database-cf55fa0176c54268b0f07242ba17b25e)



### Implementation description

- Added dependencies for Prisma and Prisma Client 
- Initialized Prisma within the project
- Created a [tutorial](https://www.notion.so/uwblueprintexecs/Set-Hook-up-database-cf55fa0176c54268b0f07242ba17b25e) to provision a database for local development



### Steps to test

1. Go through the steps in the Notion document to ensure the database can be set up correctly 



### What should reviewers focus on?

- 

### Checklist
